src/Cake.Common/Tools/DotNet/DotNetAliases.cs

```diff
@@ -1934,5 +1935,68 @@ public static IEnumerable<DotNetWorkload> DotNetWorkloadSearch(this ICakeContext
             var searcher = new DotNetWorkloadSearcher(context.FileSystem, context.Environment, context.ProcessRunner, context.Tools);
             return searcher.Search(searchString, settings);
         }
+
+        /// <summary>
+        /// Lists all installed workloads.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <returns>The list of installed workloads.</returns>
+        /// <example>
+        /// <code>
+        /// var workloadIds = DotNetWorkloadList();
+        ///
+        /// foreach (var workloadId in workloadIds)
+        /// {
+        ///      Information($"Installed Workload Id: {workloadId}");
+        /// }
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Workload")]
+        [CakeNamespaceImport("Cake.Common.Tools.DotNet.Workload.List")]
+        public static IEnumerable<string> DotNetWorkloadList(this ICakeContext context)
+        {
+            return context.DotNetWorkloadList(null);
+        }
+
+        /// <summary>
+        /// Lists all installed workloads.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <param name="settings">The settings.</param>
+        /// <returns>The list of installed workloads.</returns>
+        /// <example>
+        /// <code>
+        /// var settings = new DotNetWorkloadListSettings
+        /// {
+        ///     Verbosity = Detailed
```

Comment:
Is this intended to be?
```suggestion
        ///     Verbosity = DotNetVerbosity.Detailed
```